plugin-tailwindcss/src/mod.ts

```diff
@@ -1,20 +1,26 @@
-import type { TailwindPluginOptions } from "./types.ts";
-import { initTailwind } from "./compiler.ts";
 import type { FreshBuilder } from "fresh/dev";
 import type { App } from "fresh";
+import twPostcss from "@tailwindcss/postcss";
+import postcss from "postcss";
+import type { TailwindPluginOptions } from "./types.ts";
+
+// Re-export types for public API
+export type { TailwindPluginOptions } from "./types.ts";
 
-export function tailwind<T>(
+export async function tailwind<T>(
```

Comment:
Let's keep the sync variant of before so that we don't have to change this to be `async`.